pytest.ini

```diff
@@ -7,9 +7,6 @@ markers =
     mongo: Test the MondoDb core.
 addopts =
     # --doctest-modules
-    --cov=cachier
```

Comment:
Nope. You don't get to delete this. The coverage report to codecov.io relies on this being generated in travis.

Reply:
All three lines. :)